yarpl/include/yarpl/flowable/Flowable.h

```diff
@@ -337,6 +339,12 @@ auto Flowable<T>::take(int64_t limit) {
 }
 
 template <typename T>
+auto Flowable<T>::skip(int64_t offset) {
+  return Reference<Flowable<T>>(
```

Comment:
The way it's written now is the same for each other operator - changing it would break that convention. Do it anyways?